ramalama/model_factory.py

```diff
@@ -49,7 +49,12 @@ def detect_model_model_type(
             return Ollama, self.create_ollama
         if self.model.startswith("oci://") or self.model.startswith("docker://"):
             return OCI, self.create_oci
-        if self.model.startswith("http://") or self.model.startswith("https://") or self.model.startswith("file://"):
+        if (
+            self.model.startswith("http://")
+            or self.model.startswith("https://")
+            or self.model.startswith("file://")
+            or self.model.startswith("url://")
```

Comment:
This change looks great! My one question... Why "url://"? anything with "://" in it is a url. What is before "://" defines the protocol. A url protocol doesn't exist.

Reply:
This is basically the `model_type` which is getting passed to the store here:
https://github.com/containers/ramalama/blob/main/ramalama/model_factory.py#L86

So the protocols http, https and file get mapped to url - so we partially lose the protocol information. This seems like fixing the wrong thing when thinking about it more. Instead of passing `type(model)` to the store, it should probably be `model.type` (which is the url scheme). This would lead to models being re-downloaded again, unfortunately.